DBMS/Assignment4/zipcode.sql

```diff
@@ -0,0 +1,54 @@
+    -- Creating Database zipcodeInfo
+    CREATE DATABASE zipcodeInfo;
+    USE zipcodeInfo;
+    
+    -- creating table Zipcode
+    CREATE TABLE zipcodeNumber(
+        zipcode int PRIMARY KEY AUTO_INCREMENT
+    );
+    -- Creating table cityInfo
+    CREATE TABLE cityInfo(
+        zipcode int PRIMARY KEY AUTO_INCREMENT,
```

Comment:
ACTION: Table structure for cityInfo is not correct. If you want zipcode as primary key then why you don't create a single table to maintain city and state and zipcode, here maintaining zipcode in city and state table is not good. it is just data redundancy. if you want to implement normalization you should implement cityInfo table like that. it should have 4 columns  city_id, city_name, foreign key reference with zipcode table, foreign key with state table.